app/scripts/controllers.js

```diff
@@ -107,17 +107,34 @@ app.controller('MainController', ['$scope','$http','$location','$log', '$session
         offset = 0;
         $scope.$storage.currentPage = 1;
       }
+      // put the sorting into an enum and bool based format
+      var sortMethodDescending;
```

Comment:
can you do just

sortMethodDescending = (($scope.$storage.sortMethod % 2) == 1)

?

What is $scope.$storage.sortMethod and what do its values entail?

Reply:
I tried to do something like that initially, but I was having some issues with sending a boolean over HTTP, and sending a bool in int format was a lot easier.

$scope.$storage.sortMethod is an int variable(I'm not sure why it's in that format, I think it has to do with the variable going between different controllers), and it contains the sort method enumerations